link.ld

```diff
@@ -63,6 +61,18 @@ SECTIONS
     *(.ARM.exidx* .gnu.linkonce.armexidx.*)
     *(.debug_info)
   }
+
+  ledger.target (INFO): { KEEP(*(ledger.target)) }
```

Comment:
Removed `.` in front of custom sections for compatibility with C sdk. Also, `.` prefixes are supposed to be reserved for "system" section names.